### DISCUSSION TITLE
A 20" display for my Bambulab 3D printer

### DISCUSSION THREAD

**Discussion by kanflo** [greghesp/ha-bambulab#776] [Show and tell]
I thought the tiny display on my P1S was too small so I did a (non interactive yes, but still) 20" version;
![IMG_9759](https://github.com/user-attachments/assets/d5853b70-2613-4046-9e81-9704f183511d)

[Link to my repo](https://github.com/kanflo/pymirror-bambulab). This excellent repo is used to pull the print job cover.

**Comment by AdrianGarside** [comment]
Well, you know what they always say. 'Go big or go home!'
